.vscode/settings.json

```diff
@@ -2,7 +2,7 @@
 {
   "editor.tabSize": 2,
   "editor.insertSpaces": true,
-  "editor.formatOnSave": true,
+  // "editor.formatOnSave": true,
```

Comment:
Were these edits to settings.json intentional?

Reply:
sorry will revert...